### PR TITLE
More 64bit build cleanups

### PR DESCRIPTION
**Pull request by 6by9**
Debug messages using the wrong format specifiers again.

### CODE REVIEW

File: drivers/input/touchscreen/rpi-ft5406.c [hunk 1]
```diff
@@ -218,8 +218,8 @@ static int ft5406_probe(struct platform_device *pdev)
 
 	if (!ts->ts_base) {
 		dev_warn(dev,
-		"set failed, trying get (err:%d touchbuf:%x virt:%p bus:%x)\n",
-		err, touchbuf, ts->ts_base, ts->bus_addr);
+		"set failed, trying get (err:%d touchbuf:%x virt:%p bus:%pad)\n",
+		err, touchbuf, ts->ts_base, &ts->bus_addr);
```
Comment (popcornmix) [inline drivers/input/touchscreen/rpi-ft5406.c:222]: Isn't the contents of this more useful than the address?

Comment (pelwell) [reply drivers/input/touchscreen/rpi-ft5406.c:222]: The kernel printk physical address support expects a pointer to the address.

Comment (popcornmix) [reply drivers/input/touchscreen/rpi-ft5406.c:222]: Ok

Comment (6by9) [reply drivers/input/touchscreen/rpi-ft5406.c:222]: As Phil says, it wants a reference, which is a right pain as you can't then print "base+offset" without assigning it to a variable first. Hence ```bo->paddr + fb->offsets[0]``` being changed to ```&fbinfo->base``` (which has the same value) in a couple of places.

Docs at https://github.com/raspberrypi/linux/blob/rpi-4.19.y/Documentation/core-api/printk-formats.rst#dma-address-types-dma_addr_t
```
DMA address types dma_addr_t
----------------------------
	%pad	0x01234567 or 0x0123456789abcdef

For printing a dma_addr_t type which can vary based on build options,
regardless of the width of the CPU data path.

Passed by reference.
```

**Pull request merged**